src/components/screens/reclaimBalance/modal/status/index.js

```diff
@@ -4,6 +4,7 @@ import { transactionBroadcasted } from '@actions';
 import Status from './status';
 
 const mapStateToProps = state => ({
+  isMigrated: state.account.info?.LSK?.summary.isMigrated,
```

Comment:
If the account wasn't signed in, i.e. there was no `state.account.info.LSK` we shouldn't even see this page. The null check seems redundant.